Upstream: no
Reason: disable self-update functionality, managed via apk

--- a/cmd/update.go
+++ b/cmd/update.go
@@ -27,6 +27,9 @@ Flags:
 }

 func runUpdate(cmd *cobra.Command, args []string) error {
+	fmt.Printf("Version managed via apk: apk -U upgrade mkbrr\n")
+	return nil
+
 	_, err := semver.ParseTolerant(version)
 	if err != nil {
 		return fmt.Errorf("could not parse version: %w", err)
